src/ui/image/Image.tsx

```diff
@@ -22,12 +22,12 @@ const Image = ({ lowQuality, fallback, src, className, ...props }: Image.Props)
 
     const loadImage = () => {
```

Comment:
Add returning type def for the future update with new eslint rules

Reply:
done